ci/docker_cache.py

```diff
@@ -37,7 +37,7 @@
 DOCKERHUB_LOGIN_NUM_RETRIES = 5
 DOCKERHUB_RETRY_SECONDS = 5
 DOCKER_CACHE_NUM_RETRIES = 3
-DOCKER_CACHE_TIMEOUT_MINS = 15
+DOCKER_CACHE_TIMEOUT_MINS = 30
```

Comment:
So does this mean that even though we have a working container that has all of the deps in it, we'll still build a new one from scratch after 15 (or now 30) minutes?

If this is the case, why isn't this number much, much higher? Like in terms of hours or days? I am always frustrated by watching the same intermediate dependencies being built when there doesn't seem to be any change at all.

Reply:
No, this is the job that publishes the cache. Right now, the job fails because it runs into a timeout. Once the job works again, everything will be back to normal with incremental builds.

Why shall we set it way higher? The job is only supposed to run a few minutes

Reply:
Nevermind then, I thought this would explain why I see the cache invalidated so often and for no apparent reason.

Reply:
Yeah the reason for the cache invalidations is because this job is broken since the 20th of September.